airflow-core/docs/core-concepts/auth-manager/simple.rst

```diff
@@ -18,12 +18,9 @@
 Simple auth manager
 ===================
 
-.. note::
-    Before reading this, you should be familiar with the concept of auth manager.
-    See :doc:`/core-concepts/auth-manager/index`.
-
 .. warning::
-  The simple auth manager is intended to be used for development and testing purposes. It should not be used in a production environment.
+  The Simple auth manager is intended for development and testing. If you're using it in production, ensure that
+  access is controlled through other means.
```

Comment:
I would add a note mentioning how to disable this warning.

Reply:
I updated the description of the PR to include a screenshot. I think it is pretty straightforward how to disable this warning (click on the "X").

Reply:
oh -- is it then gone forever?
or does it come back after image deploy? or next time you visit?

Reply:
(there's a difference between dismissing and disabling)

Reply:
oh it's only on the login page -- that's less of a problem to me

Reply:
is there a way to have no auth at all?  like just straight to home page?

Reply:
When user clicks on it, I save it in local storage (browser) so the message will be gone as long as the user uses the same browser and does not clear its local storage

Reply:
> is there a way to have no auth at all? like just straight to home page?

Yes, there is an option with simple auth manager. If you set `[core] simple_auth_manager_all_admins` to `True`, then users are automatically logged in with admin permissions

Reply:
nice.